operatorcert/entrypoints/publish.py

```diff
@@ -150,6 +150,8 @@ def create_repository(args: Any, project: Dict[str, Any]) -> Any:
     repository = f"{vendor_label}/{repo_name}"
 
     long_description = container.get("repository_description", " ")
```

Comment:
```suggestion
    long_description = container.get("repository_description") or " "
```
This will do the same thing without the if statement.